devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -441,8 +442,6 @@ def configure() {
 				zigbee.configureReporting(0xFC02, 0x0013, DataType.INT16, 1, 3600, 0x0001, [mfgCode: manufacturerCode]) +
 				zigbee.configureReporting(0xFC02, 0x0014, DataType.INT16, 1, 3600, 0x0001, [mfgCode: manufacturerCode])
 	}
-	configCmds += zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, batteryAttr)
```

Comment:
We need to read the battery a second time. This was added because we weren't always getting the correct value the first time. I'd have to look through history to remind myself of the exact issue but I know this was done on purpose.